mkdocs/docs/experiments/categories/pods/pod-dns-error/target-hostnames.yaml

```diff
@@ -12,13 +12,13 @@ spec:
     appkind: "deployment"
   chaosServiceAccount: pod-dns-error-sa
   experiments:
-  - name: pod-dns-error
-    spec:
-      components:
-        env:
-        ## comma separated list of host names
-        ## if not provided, all hostnames/domains will be targeted
-        - name: TARGET_HOSTNAMES
-          value: '["litmuschaos","chaosnative.com"]'
-        - name: TOTAL_CHAOS_DURATION
-          value: '60'
+    - name: pod-dns-error
+      spec:
+        components:
+          env:
+            ## comma separated list of host names
+            ## if not provided, all hostnames/domains will be targeted
+            - name: TARGET_HOSTNAMES
+              value: '["litmuschaos"]'
+            - name: TOTAL_CHAOS_DURATION
+              value: "60"
```

Comment:
Can you check the below github action warnings?

Reply:
same :)